_redirects

```diff
@@ -99,7 +99,7 @@
 /nj2021 https://www.theaterdeveste.nl/programma/7889/Krashna_Musika/Najaarsconcert_2021 301!
 
 # Verdi Concertgebouw 2022 shortener
-/verdi https://www.concertgebouw.nl/concerten/592421-verdis-requiem-door-krashna-musika-en-het-vu-koor 301!
+/verdi /concerts/verdi-2022-amsterdam 301!
```

Comment:
Why change this? Is it not better to redirect people directly to the place where they can buy tickets? Saves them a click :)

EDIT: Never mind my comment. I see it's because of COVID uncertainties!